lib/steep/services/signature_help_provider.rb

```diff
@@ -11,15 +11,10 @@ def env
         subtyping.factory.env
       end
 
-      def initialize(source:, subtyping:)
+      def initialize(source:, content:, subtyping:)
         @source = source
         @subtyping = subtyping
-
-        text =
-          if source.node
-            source.node.loc.expression.source
-          end
-        @buffer = RBS::Buffer.new(name: source.path, content: text || "")
+        @buffer = RBS::Buffer.new(name: source.path, content: content)
```

Comment:
It looks like we can simply use `Source#buffer` here.

```suggestion
        @buffer = source.buffer
```

Reply:
`Source#buffer` was added very recently, weeks ago.

Reply:
Great! It works fine!